ViewPumpSample/src/main/java/io/github/inflationx/viewpump/sample/CustomViewWithTypefaceSupport.java

```diff
@@ -0,0 +1,75 @@
+package io.github.inflationx.viewpump.sample;
+
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Rect;
+import android.graphics.Typeface;
+import android.os.Build;
+import android.util.AttributeSet;
+import android.view.View;
+
+/**
+ * @author Dmitriy Tarasov
+ */
+public class CustomViewWithTypefaceSupport extends View {
```

Comment:
Think the ViewPump sample should remove most of the Calligraphy stuff and focus on simpler `ViewPump` examples.